components/brave_new_tab_ui/components/default/brandedWallpaper/logo.tsx

```diff
@@ -11,13 +11,14 @@ import * as Styled from './logo-style'
 
 interface Props {
   data: NewTab.BrandedWallpaperLogo
+  onClickLogo: () => void
 }
 
-function Logo ({ data }: Props) {
+function Logo ({ data, onClickLogo }: Props) {
   return (
     <>
       <Styled.Image src={data.image} alt={data.alt} />
-      <Styled.Anchor href={data.destinationUrl} title={data.alt}>
+      <Styled.Anchor title={data.alt} onClick={onClickLogo}>
```

Comment:
@petemill Can you check this change? With this, link click is delegated to browser.

Reply:
NVM - `href={data.destinationUrl}` change is reverted  and only `onClick` is added.